code/week5/prey.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <iostream>
```

Comment:
tabs instead of spaces (:cry:)

(actually a bunch of files have this, so I'm going to just leave this one)

[install a plugin](http://editorconfig.org/#download) to avoid worrying about this in the future...